boa_engine/src/job.rs

```diff
@@ -0,0 +1,39 @@
+use crate::{Context, JsResult, JsValue};
+
+use gc::{Finalize, Trace};
+
+#[derive(Debug, Clone, Trace, Finalize)]
+pub struct JobCallback {
```

Comment:
I would like to have a bit of documentation in this structure, to understand it better.